pkgs/development/compilers/chicken/5/overrides.nix

```diff
@@ -54,8 +58,12 @@ in {
   sdl2-ttf = addToBuildInputs pkgs.SDL2_ttf;
   soil = addToPropagatedBuildInputsWithPkgConfig pkgs.libepoxy;
   sqlite3 = addToBuildInputs pkgs.sqlite;
-  stemmer = old: (brokenOnDarwin old) // (addToBuildInputs pkgs.libstemmer old);
-  stfl = addToBuildInputs [ pkgs.ncurses pkgs.stfl ];
+  stemmer = old:
+    (addToBuildInputs pkgs.libstemmer old)
+    // (addToCscOptions [ "-L -lstemmer" ] old);
```

Comment:
Maybe there are linking flags propagated from `pkgs.libstemmer` that we are just not handing off to the chicken build. It would be unfortunate if we have to maintain all these linking flags for each package when I think that they are already there somewhere. Then we could get rid of `CSC_OPTIONS` altogether.

Reply:
If we decide to go with this for the short term the string should start with a space like so: `" -L -lstemmer"`.

Reply:
`toString` is basically `lib.concatStringsSep " "`, so the leading space is optional.